docs/markdown/Cross-compilation.md

```diff
@@ -153,7 +154,8 @@ single string (i.e. not as `'-DCROSS=1 -DSOMETHING=3'`).
 system path (the system that will run the compiled binaries). This is used
 internally by Meson to set the PKG_CONFIG_SYSROOT_DIR environment variable
 for pkg-config. If this is unset the host system is assumed to share a root
-with the build system.
+with the build system. The 'pkg_config_libdir' property may point to a list
+of path used by pkg-config to get the cross build dependencies.
```

Comment:
I would have that as new paragraph, something lile:

> *Since 0.54.0* The `pkg_config_libdir` property may point to a list of path used internally by Meson to set the PKG_CONFIG_LIBDIR environment variable for pkg-config. This prevents pkg-config from searching cross dependencies in system directories.